types/clone/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for clone 0.1.11
+// Type definitions for clone 2.1.2
```

Comment:
```suggestion
// Type definitions for clone 2.1
```
https://github.com/definitelyTyped/DefinitelyTyped/#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

Reply:
Thanks, will keep that in mind next time!